ironfish/src/network/peers/connections/webRtcConnection.ts

```diff
@@ -59,7 +59,7 @@ export class WebRtcConnection extends Connection {
     initiator: boolean,
     logger: Logger,
     metrics?: MetricsMonitor,
-    options: { simulateLatency?: number } = {},
+    options: { simulateLatency?: number; stunServers: string[] } = { stunServers: [] },
```

Comment:
For this we should just make it optional. You shouldn't materialize these defaults unless you actually need the real value.

Reply:
I thought providing a default value made it optional. I can change though

Reply:
yeah it does. This is just more of a nit because we don't do defaulting values like this in other places, we just materialize it where we use it, or in the constructor. To be clear, I think this code is fine, I'm just trying to massage it to be more inline with other code.